Feature selection

sahas111 opened the pull request:
I have uploaded here the latest feature_selection_analysis script for your review.

sahas222 committed:
Feature_selection_analysis script

Hopefully I have correctly uploaded it now

sahas222 committed:
feature_selection_analysis script

Hopefully I have correctly uploaded the script
this time.

jni commented:
Looks like the PR is well submitted! =) I'll add some comments now.

jni commented on feature_selection_analysis.py:
```diff
@@ -0,0 +1,118 @@
+
+    final_dataframe.index = names
+    return final_dataframe
+
```
A better way to include the commands below is to put them in a `__main__` guard:

``` python
if __name__ == '__main__':
    filenames = glob.glob('*.tif')
    [...]
```

Then you can just do:

```
$ python feature_selection_analysis.py
```

in the right working directory, and it will run the analysis! You just need to make sure that the `final_data` is saved. =)

jni commented on feature_selection_analysis.py:
```diff
@@ -0,0 +1,118 @@
+     image thresholding or the reduction of a graylevel image to a binary image.
+    """
+
+    names = ['original', 'cells-with', 'frac', 'd-neighbor', 'theta', 'cos', 'sin', 'adaptive', 'solidity', 'intensity', 'extent', 'euler_number', 'eccentricity', 'area', 'otsu-threshold']
```
You should make sure lines don't go over 80 characters.

jni replied:
look for "continuation" in PEP8

jni replied:
This line needs to be cut in half (as we discussed for your other PR)

sahas222 committed:
Executable as a main function

Added some commands that will be excecuted
when the module will be run as a program.
Fixed the continuation problem in the docstring.
I have to change my own directory and module
organisation now to run the module directly
which I will do after consulting with you.

jni commented on feature_selection_analysis.py:
```diff
@@ -0,0 +1,138 @@
+    file_names = glob.glob('*.tif')
+    df = dataframe(file_names)
+    df.to_csv('dataframe1.csv')
+    final_data = intra_vs_inter_gene_dist_hist(df)
```
Add one more line to save final_data to a csv also

jni commented:
Almost ready, @sahas111! I've added two more minor comments, then I will merge this.

sahas222 committed:
Fixed Continuation error

Fixed the line and added a command for
saving the final data to a csv file too.
